app/ui/content/logging.js

```diff
@@ -0,0 +1,22 @@
+/*
+ Copyright 2016 Mozilla
+
+ Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+ this file except in compliance with the License. You may obtain a copy of the
+ License at http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software distributed
+ under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ specific language governing permissions and limitations under the License.
+ */
+
+import bunyan from 'bunyan';
+import { pipeToConsole } from '../../shared/logging';
+
+export const logger = bunyan.createLogger({
+  name: 'content',
+  streams: [
+    // TODO pipe this to the main process stdout
```

Comment:
I have no idea, it doesn't on Windows I think but stdout is almost completely busted on windows, partly why I'm doing this to make it actually possible to fix.